examples/Spirit/spirit_utils.h

```diff
@@ -62,9 +87,95 @@ std::tuple<  std::vector<std::unique_ptr<dairlib::systems::trajectory_optimizati
+          std::vector<double> mus );
+
+
 
-}
+/// This overload sets an individual joint's position limit
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+///    @param iJoint the integer index of the joint
+///    @param minVal joint's minimum position
+///    @param minVal joint's maximum position
+template <typename T>
+void setSpiritJointLimits(
+                    drake::multibody::MultibodyPlant<T> & plant, 
+                    dairlib::systems::trajectory_optimization::Dircon<T>& trajopt,  
+                    int iJoint, 
+                    double minVal, 
+                    double maxVal  );
+
+/// This overload sets an a set of joints' position limits
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+///    @param iJoints the integer indices (vector) of the joints to be limited
+///    @param minVals vector of joints' minimum positions
+///    @param minVals vector of joints' maximum positions
+template <typename T>
+void setSpiritJointLimits(
+                    drake::multibody::MultibodyPlant<T> & plant, 
+                    dairlib::systems::trajectory_optimization::Dircon<T>& trajopt,  
+                    std::vector<int> iJoints, 
+                    std::vector<double> minVals, 
+                    std::vector<double> maxVals  );
+
+/// This overload sets an a set of joints' position limits to the same thing
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+///    @param iJoints the integer indices (vector) of the joints to be limited
+///    @param minVal joints' minimum position
+///    @param minVal joints' maximum position
+template <typename T>
+void setSpiritJointLimits(
+                    drake::multibody::MultibodyPlant<T> & plant, 
+                    dairlib::systems::trajectory_optimization::Dircon<T>& trajopt,  
+                    std::vector<int> iJoints, 
+                    double minVal, 
+                    double maxVal  );
+
+/// This overload sets all the joints to their nominal limit's
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+template <typename T>
+void setSpiritJointLimits(
+                    drake::multibody::MultibodyPlant<T> & plant, 
+                    dairlib::systems::trajectory_optimization::Dircon<T>& trajopt );
+
+
+/// Sets all the joints' actuator limits to the same thing
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+///    @param actuatorLimit the (symmetric) effort limit 
+template <typename T> 
+void setSpiritActuationLimits(
+          drake::multibody::MultibodyPlant<T> & plant, 
+          dairlib::systems::trajectory_optimization::Dircon<T>& trajopt,
+          double actuatorLimit = 40.0);//From URDF this default seems a bit high
```

Comment:
What would be the actual correct value? I would assume 3.5*gear ratio would be fairly accurate